README.md

```diff
@@ -76,17 +78,17 @@ Currently, [Azure Key Vault plugin for Notation](https://github.com/Azure/notati
     plugin_config: |-
       ca_certs=.github/cert-bundle/cert-bundle.crt
       self_signed=false
+    timestamp_url: http://timestamp.digicert.com
+    timestamp_root_cert: ./sign/tsaRootCert/DigiCertTSARootSHA384.cer
```

Comment:
Should this cert be put somewhere under `.github`?

Reply:
Updated.